packages/deployer/tasks/create-dummy-imc.js

```diff
@@ -0,0 +1,28 @@
+const { task } = require('hardhat/config');
```

Comment:
This task creates a fake GenIMCMixin.sol contract with random addresses for all the modules available to allow the developer to compile solidity source without errors